### PR TITLE
 bump versions of used CNAO and kubevirt 

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
cc @RamLavi

File: hack/deploy-test-cluster.sh [hunk 1]
```diff
@@ -6,15 +6,27 @@ set -ex
...
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/namespace.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/network-addons-config.crd.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/operator.yaml
-./cluster/dind-cluster/kubectl create -f ./hack/cna/cna-cr.yaml
-
-# Deploy Kubevirt
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/v0.20.4/kubevirt-operator.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/v0.20.4/kubevirt-cr.yaml
+
+echo 'Deploying Linux bridge CNI and Multus ...'
```
Comment (phoracek) [inline hack/deploy-test-cluster.sh]: Please note that this will go away as soon as we move to prow (we will drop dind)

File: cluster/up.sh [hunk 2]
```diff
@@ -21,37 +21,25 @@ kubevirtci::install
...
+echo 'Deploying Linux bridge CNI and Multus ...'
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/namespace.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/network-addons-config.crd.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/operator.yaml
+cat <<EOF | ./cluster/kubectl.sh create -f -
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
```
Comment (RamLavi) [inline cluster/up.sh]: In the original hack/cna/cna-cr.yaml we also deploy imagePullPolicy. why did you drop it?

File: hack/cni-plugins/cni-plugins.yaml [hunk 3]
```diff
@@ -1,43 +0,0 @@
----
```
Comment (RamLavi) [inline hack/cni-plugins/cni-plugins.yaml]: why are we dropping this yaml?

File: hack/multus/kubernetes-multus.yaml [hunk 4]
```diff
@@ -1,25 +0,0 @@
----
-kind: ConfigMap
```
Comment (RamLavi) [inline hack/multus/kubernetes-multus.yaml]: why are we dropping hack/multus yamls?

File: cluster/up.sh (hunk 2 above)
Comment (phoracek) [reply cluster/up.sh]: We don't need to pull it Always, we are ok with whatever Multus and bridge CNI is given to us. And OnlyIfNeeded is the default.

File: hack/cni-plugins/cni-plugins.yaml (hunk 3 above)
Comment (phoracek) [reply hack/cni-plugins/cni-plugins.yaml]: CNI plugins are shipped by CNAO, this is just a remnant of the past.

File: hack/multus/kubernetes-multus.yaml (hunk 4 above)
Comment (phoracek) [reply hack/multus/kubernetes-multus.yaml]: Multus shipped by CNAO, this is just a remnant of the past.

File: cluster/up.sh [hunk 5]
```diff
@@ -21,37 +21,25 @@ kubevirtci::install
...
 $(kubevirtci::path)/cluster-up/up.sh
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
-
-# wait for cluster operator
+echo 'Deploying Linux bridge CNI and Multus ...'
```
Comment (RamLavi) [inline cluster/up.sh]: why do we need multus and linux bridge deployed? is it so we have "customers" to give MACs to?

File: cluster/up.sh [hunk 6]
```diff
@@ -21,37 +21,25 @@ kubevirtci::install
...
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/namespace.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/network-addons-config.crd.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/operator.yaml
+cat <<EOF | ./cluster/kubectl.sh create -f -
```
Comment (RamLavi) [inline cluster/up.sh]: I noticed that you generally change from using yaml files to heredocs, could you explain why you prefer this?

File: cluster/up.sh (hunk 5 above)
Comment (phoracek) [reply cluster/up.sh]: Yes. In e2e tests we connect Pods and VMs to secondary networks via Multus and the bridge CNI.

File: cluster/up.sh (hunk 6 above)
Comment (phoracek) [reply cluster/up.sh]: Less jumps between files. Once we drop dind, all the dependencies would be here. But if you prefer it being in a separate file, I don't mind that much.

**Pull request closed without merging**